### DISCUSSION TITLE
[Feat]: Add third option to forceWheelAxis for both ways navigation

### DISCUSSION THREAD

**Discussion by psntr** [davidjerleke/embla-carousel#1094] [Ideas]
### Which variants of Embla Carousel are relevant to this feature request?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Currently the plugin [Wheel Gestures](https://www.embla-carousel.com/plugins/wheel-gestures/) allows either `x` or `y`
 direction to navigate the carousel, but it would be great to allow a third option which is to allow both directions. 

### Additional Context

Facing an issue where I'm displaying a carousel full bleed and wish to allow the user to use the wheel pad or wheel mouse to navigate the carousel by either scrolling up/down or left/right. 

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by sarussss** [comment]
Hi @psntr 
This question needs to be posted [here](https://github.com/xiel/embla-carousel-wheel-gestures) because it was developed by Felix Leupold.
